src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -289,11 +289,37 @@ class CompilerContainer {
            (version.includes('nightly') && this._view.includeNightlies.checked)
   }
 
-  _updateVersionSelector () {
+  _updateVersionSelector (pragma = null) {
```

Comment:
Maybe add a comment with the type of object you want to get : 
```javascript
/**
 * Update ***
 * @params pragma {Object} Version of solidity compiler
*/
```
Here is a reference to [@param tag](https://devdocs.io/jsdoc/tags-param)
Even better would be to use the [@type tag](https://devdocs.io/jsdoc/tags-type) so we know how the object should look like.